calculator.go

```diff
@@ -1,13 +1,94 @@
-// Package calculator provides a library for simple calculations in Go.
 package calculator
 
-// Add takes two numbers and returns the result of adding them together.
-func Add(a, b float64) float64 {
-return a + b
+import (
+	"errors"
+	"math"
+	"strconv"
+	"strings"
+)
+
+// Add takes a variable number of float64 and returns their sum.
+func Add(nums ...float64) float64 {
```

Comment:
This is good, but one potential problem is that we could pass _zero_ arguments to Add, which isn't meaningful (neither is passing one argument). Is there a way to get the compiler to make sure that Add gets at least two arguments?